ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -285,6 +294,12 @@ Item {
 
                         onOpenGifPopupRequest: root.openGifPopupRequest(params, cbOnGifSelected, cbOnClose)
 
+                        // Contacts related requests:
+                        onChangeContactNicknameRequest: root.changeContactNicknameRequest(pubKey, nickname, displayName, isEdit)
```

Comment:
basically qt 6 forces to use function-based syntax:
```
onChangeContactNicknameRequest: (pubKey, nickname, displayName, isEdit) => {
  root.changeContactNicknameRequest(pubKey, nickname, displayName, isEdit)
}
```

otherwise it prints warnings.

Alternative solution is to forward signals in `Component.onCompleted`:

```
Component.onCompleted: {
  removeTrustStatusRequest.connect(root.removeTrustStatusRequest)
}
```